config/management/src/json_rpc.rs

```diff
@@ -31,15 +33,39 @@ impl JsonRpcClientWrapper {
             .map_err(|e| Error::JsonRpcReadError("account-state", e.to_string()))
     }
 
-    pub fn get_sequence_number(&self, account: AccountAddress) -> Result<u64, Error> {
+    pub fn get_validator_config(&self, account: AccountAddress) -> Result<ValidatorConfig, Error> {
```

Comment:
is there any chance of code reuse between key manager and this (secure/key-manager), okay if we want to address that later.

Reply:
Yeah, I think there is a lot of reuse that could happen between the `JsonRpcLibraInterface` and this wrapper.  I was focusing on getting the stuff in a working state, and the next PR would be to pull that key manager reuse.

I will say key manager's stuff is pretty straightforward.